content/index.pl.md

```diff
@@ -0,0 +1,144 @@
+-   **twórcy** zapewniają odpowiednie sformułowania tekstu alternatywnego
+-   **narzędzia autorskie** umożliwiają, ułatwiają i promują zamieszczanie tekstu alternatywnego na stronie internetowej
+-   **narzędzia oceny** służą do sprawdzenia, czy istnieje tekst alternatywny
+-   **oprogramowanie użytkownika** zapewnia ludziom i maszynom interfejs do obsługi tekstu alternatywnego w&nbsp;różnych trybach
+-   **użytkownicy** wiedzą jak odczytać tekst alternatywny w&nbsp;wykorzystywanym przez nich oprogramowaniu i/lub technologiach wspomagających, w razie potrzeby.
+	  
+### Cykl implementacji
+
+Gdy funkcje dostępności zostaną skutecznie zaimplementowane w jednym komponencie, istnieje większe prawdopodobieństwo, że zostaną wdrożone w innych komponentach.
+
+{% include image.html src="cycle.pl.png" alt="" %}
+
+- Gdy **przeglądarki internetowe, odtwarzacze mediów, technologie wspomagające** i inne **programy użytkownika** obsługują funkcje dostępności, użytkownicy częściej będą tego wymagać, a programiści częściej implementować je w swoich **treściach**.
+- Gdy twórcy chcą wdrażać funkcje dostępności w swoich **treściach**, częściej domagają się, aby **narzędzie autorskie** ułatwiało ich wdrożenie. 
+- Gdy **narzędzia autorskie** umożliwiają łatwą implementację funkcji dostępności, twórcy chętniej je wdrożą w swoich **treściach**. 
+- Gdy funkcje dostępności są zaimplementowane w **większości treści**, twórcy i&nbsp;użytkownicy częściej żądają, aby obsługiwały je **programy użytkownika**.
+
+### Gdy jeden komponent jest słaby
+
+Gdy funkcje dostępności nie są zaimplementowane w którymś z komponentów, motywacja do zaimplementowania ich w&nbsp;pozostałych komponentach spada, jeśli nie skutkuje to zwiększeniem dostępności dla użytkownika. Na przykład mało prawdopodobne jest, aby programiści wdrożyli funkcję dostępności, która nie jest obsługiwana przez narzędzia autorskie i której nie obsługuje większość przeglądarek lub technologii wspomagających.
+
+{% assign longdesc_url = "/fundamentals/components/examples/pl#weak" | relative_url %}
+{% include image.html src="bridge.pl.png" alt="Ilustruje, co dzieje się, gdy jeden komponent jest słaby" longdesc=longdesc_url %}
+
+Gdy jeden komponent kiepsko obsługuje dostępność, inne komponenty mogą to czasami zrekompensować poprzez „obejścia”, choć wymaga to znacznie większego wysiłku i ogólnie nie jest dobre dla dostępności. Na przykład:
+
+-   twórcy mogą wykonać więcej pracy, aby zrekompensować brak obsługi dostępności w&nbsp;narzędziach autorskich; na przykład poprzez bezpośrednie kodowanie znaczników zamiast za pomocą narzędzia,
+-   użytkownicy mogą wykonać więcej pracy, aby zrekompensować  brak obsługi dostępności w przeglądarkach, odtwarzaczach mediów lub technologiach wspomagających; na przykład, używając innej przeglądarki lub technologii wspomagającej, aby przezwyciężyć różne problemy z dostępnością.
```

Comment:
@zwiastunsw A paragraph from the original version is missing in the translation:

> However, in most cases the works-arounds are not implemented and the result is still poor accessibility. Additionally, sometimes poor accessibility support in one component cannot be reasonably overcome by other components and the result is inaccessibility, making it impossible for some people with disabilities to use a particular website, page, or feature.